include/v4l2_camera/rate_bound_status.hpp

```diff
@@ -0,0 +1,281 @@
+// Copyright 2025 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef RATE_BOUND_STATUS_HPP_
+#define RATE_BOUND_STATUS_HPP_
+
+#include <diagnostic_updater/diagnostic_updater.hpp>
+#include <diagnostic_msgs/msg/diagnostic_status.hpp>
+
+#include <iomanip>
+#include <sstream>
+#include <variant>
+#include <optional>
+#include <mutex>
+#include <stdexcept>
+#include <chrono>
```

Comment:
The headers are not sorted and include-what-you-use flags the `string` and `limits` headers missing. Below is how pre-commit CI in Nebula changes it:

```suggestion
#include <diagnostic_updater/diagnostic_updater.hpp>

#include <diagnostic_msgs/msg/diagnostic_status.hpp>

#include <chrono>
#include <iomanip>
#include <limits>
#include <mutex>
#include <optional>
#include <sstream>
#include <stdexcept>
#include <string>
#include <variant>
```